packages/app-store/exchange2013calendar/api/add.ts

```diff
@@ -49,7 +50,7 @@ async function postHandler(req: NextApiRequest, res: NextApiResponse) {
     return res.status(500).json({ message: "Could not add this exchange account" });
   }
 
-  return { url: "/apps/installed" };
+  return { url: getInstalledAppPath({ variant: "calendar", slug: "exchange2013-calendar" }) };
```

Comment:
This is to support redirecting to the correct category within the Installed Apps page once the app has been installed.